dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/DeepStorageTaskProgressViewSkin.java

```diff
@@ -177,20 +178,22 @@ protected void updateItem(final T task, final boolean empty) {
     }
 
     private void hideTask(final T task, final TaskCell tc) {
-        if (task != null && task instanceof JobTask) {
+        if (task instanceof JobTask && task != null) {
```

Comment:
any particular reason the order was changed?

Reply:
This should be more correct, we don't care if it's non-null if it isn't a JobTask.

Reply:
philosophically, can't we equally say that we don't care if its a JobTask if its null?
FYI: the code is fine, this is more just my curiosity

Reply:
Don't we care that it's not null first?  It's possible that just doing the instanceof is all that's needed if instanceof returns false if task is null.

Reply:
Looks like your right, `instanceof` has an explicit not null check built in https://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.20.2

Reply:
Ok, cool I'll remove the extra null check.